validator/validator-main.protoascii

```diff
@@ -993,6 +993,26 @@ tags: {
     dispatch_key: NAME_VALUE_DISPATCH
   }
 }
+# AMP metadata, name=amp-vars-*
+# Allows advertisers to pass information from creatives
+# to be included in amp-ad-metadata.
+tags: {
+  html_format: AMP4ADS
+  tag_name: "META"
+  spec_name: "meta name=amp-vars-*"
+  mandatory_parent: "HEAD"
+  unique: true
```

Comment:
The PR description mentions "These new meta tag**s**". Do you expect more than one of these on the same document? If so, I would not set `unique` here.

At the risk of bikeshedding (I have no strong opinion), if you expect multiple it might make sense instead to structure this like viewport, using commas to separate multiple values:

```
<meta name=amp-vars value="foo=bar;baz=lemur">
```

Reply:
hmm, this looks interesting too. although there will be some escaping needed for `=` & `;` (which is probably fine).

BTW, given we have `amp4ads-ids`, we should call this `amp4ads-vars`

Reply:
Talked offline. Our partner teams prefer multiple tags, and since we don't feel strongly we decided to go with that.

Removed `unique`, changed to `amp4ads-vars-*`